src/graphql/query/lendByCategory/lendByCategory.graphql

```diff
@@ -4,30 +4,6 @@ query lendByCategory($basketId: String) {
 			key
 			value
 		}
-		categoryDescriptionExperiment: uiExperimentSetting(key: "category_description") {
-			key
-			value
-		}
-		addToBasketPopup: uiExperimentSetting(key: "add_to_basket_v2") {
-			key
-			value
-		}
-		favoriteCountryRow: uiExperimentSetting(key: "favorite_country") {
-			key
-			value
-		}
-		mlServiceBandit: uiExperimentSetting(key: "EXP-ML-Service-Bandit-LendByCategory") {
```

Comment:
I thought this was supposed to be evergreen? That we would also be passing the distribution and header so they could test variations on the algo?